cordova/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "betaflight-configurator",
+  "displayName": "Betaflight Configurator",
+  "version": "10.7.0-RC1",
```

Comment:
@McGiverGim we don't have to edit this package.json because some values like the version will be updated automatically by the gulp task (version in `cordova/package` will be the same as in `package.json`). So It could be better to remove this values and to let the gulp task adding them automatically ?

Reply:
True, I have found it in the `gulpfile.js`. Maybe we can use the same approach for the other places where we have the version but one.

Maybe something here like:
```json
"version": "[INJECTED_BY_GULPFILE]".
```
For this and the others elements injected will make things clearer if it has no drawbacks.

Reply:
When I say that we can use it for others places, I'm not talking in this PR, is only an idea for a future PR for me or others...